src/libs/actions/BankAccounts.js

```diff
@@ -906,6 +906,22 @@ function validateRoutingNumber(number) {
     return false;
 }
 
+function getOAuthReceivedRedirectURI() {
+    if (!window.location) {
```

Comment:
When will this not be defined? Would it be better to have this method return `null` on those platforms?

Reply:
Yeah good point, was just thinking of doing something like your `getPlaidLinkTokenParams`